drivers/dma/dma_mcux_lpc.c

```diff
@@ -491,6 +558,11 @@ static int dma_mcux_lpc_init(const struct device *dev)
 
 	memset(data->data_cb, 0, size_channel_data);
 
+	/* Indicate that the Otrig Muxes are not connected */
+	for (int i = 0; i < ARRAY_SIZE(data->linked_otrig_ch); ++i) {
```

Comment:
why ++i? wouldnt this cause the loop to be 1-4 instead of 0-3?

Reply:
changed ++i to i++